### PR TITLE
modify docs/Build-for-Linux.md. Because there is wrong build option.

### PR DESCRIPTION
**Pull request by hs0225**
IoT.js-DCO-1.0-Signed-off-by: Hosung Kim hs852.kim@samsung.com

### CODE REVIEW

**Comment by yichoi** [comment]
LGTM

**Comment by LaszloLango** [comment]
@hs0225 could you fix all of them based on `tools/buildpy --help`? I'm afraid that most of them are wrong in this MD file.

File: docs/Build-for-Linux.md [hunk 1]
```diff
@@ -75,9 +75,9 @@ Options that may need explanations.
 * buildlib: generating _iotjs_ to a library if True(e.g. for NuttX). give __--buildlib__ to make it True.
 * nuttx-home: it's NuttX platform specific, to tell where the NuttX configuration and header files are.
 * init-submodule: for normal cases you can ignore this, it will checkout matching revision of each sub-module library every time. but if you want to change something or checkout other revision, use __--noinit-submodule__.
```
Comment (LaszloLango) [inline docs/Build-for-Linux.md]: `--noinit-submodule` -> `--no-init-submodule`

**Comment by hs0225** [comment]
Ok. I try to fix them.

File: docs/Build-for-Linux.md (hunk 1 above)
Comment (hs0225) [inline docs/Build-for-Linux.md]: Ok. I try to fix them.

**Comment by hs0225** [comment]
@LaszloLango I updated. Pelase review.

**Comment by LaszloLango** [comment]
@hs0225, thanks for the update. LGTM

**Comment by yichoi** [comment]
LGTM

**Pull request merged**